//离开公会
message DceGuildQuit
{
}
